tests/commands/test_export.py

```diff
@@ -0,0 +1,16 @@
+from click.testing import CliRunner
+
+from loqusdb.commands.cli import cli as base_command
+
+def test_export_base(real_db_name):
+    """Test the base command that exports variants."""
+
+    runner = CliRunner()
+
+    # WHEN the base command to export cases is run
+    command = ["--database", real_db_name, "export"]
+
+    ## THEN it should return success
+    result = runner.invoke(base_command, command)
+    assert result.exit_code == 0
```

Comment:
Can we also test that it actually exported a file?

Reply:
If you don't provide the -o param in the cli it's not exporting to file, but printing into the terminal:

<img width="1050" alt="image" src="https://github.com/Clinical-Genomics/loqusdb/assets/28093618/c635f420-885d-4c9b-b6ff-693bf79d1815">

For the purpose of testing that the are 0 cases with the modified count command I think this is good enough. 
But I agree that more and better tests for the cli could have been written!